code/__HELPERS/atoms.dm

```diff
@@ -379,3 +379,23 @@ rough example of the "cone" made by the 3 dirs checked
 		"x" = icon_width > ICON_SIZE_X && pixel_x != 0 ? (icon_width - ICON_SIZE_X) * 0.5 : 0,
 		"y" = icon_height > ICON_SIZE_Y && pixel_y != 0 ? (icon_height - ICON_SIZE_Y) * 0.5 : 0,
 	)
+
+/// Returns the living mob that is currently holding us if we are either in their inventory or a backpack analogue.
+/// Returns null if it's in an invalid location, so that we can check explicitly for null later.
+/// get_last_mob = TRUE will return the final mob, while FALSE will return the first. Only there in the snowflake case that we're in nested mobs.
+/atom/movable/proc/get_held_mob(get_last_mob = FALSE)
```

Comment:
We do kinda already have a generic proc for this, `get_atom_on_turf`.

Reply:
Wouldn't that return all mobs that are on the turf?
So if there's one person sitting underneath another person wouldn't it also return them?